lib/converter.js

```diff
@@ -75,8 +91,19 @@ async function convert(inputFile, options, timings) {
     if (format) {
       pdfOptions.format = format // Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.
     }
-    return await page.pdf(pdfOptions)
+    await page.pdf(pdfOptions)
   } finally {
+    if (outputToStdout && await existsFile(outputFile)) {
+      console.log(await readFile(outputFile, 'utf-8'))
+      await removeFile(outputFile)
```

Comment:
Curious: does JavaScript have some kind of try-with-resources (as in Java) / with (as in Python) analogue? As written, there's a theoretical possibility that `readFile` throws, and `outFile` doesn't get removed.

Reply:
Also, it seems like we should close browser in finally as well?